app/celery/provider_tasks.py

```diff
@@ -58,7 +58,9 @@ def deliver_email(self, notification_id):
             raise NoResultFound()
         send_to_providers.send_email_to_provider(notification)
     except InvalidEmailError as e:
-        current_app.logger.exception(e)
+        current_app.logger.info(
+            f"Cannot send notification {notification_id}, got an invalid email address: {str(e)}."
```

Comment:
does this stack trace have enough information to debug the problem (email address, etc.)?

Reply:
Yup, the source `InvalidEmailError` at the end of the message includes the email address and the message

https://github.com/cds-snc/notification-api/blob/a42942bb59a13cbdba074f5f380cd5d50e340d38/app/clients/email/aws_ses.py#L106-L110